### DISCUSSION TITLE
Save filter settings in grid view?

### DISCUSSION THREAD

**Discussion by Stephan9283** [pimcore/pimcore#14023] [Q&A / Support] [answered]
Hello,
I have saved a gried view and share it with others. That works fine. But I also want to apply a filter to a select field in the grid view. But these settings are  not saved. How can I do that and share it with others? Is this possible?
regards
Stephan

**Comment by SamyMP** [comment, marked as answer]
I answered in [this post](https://github.com/pimcore/pimcore/discussions/13999#discussioncomment-4630590)

It is available in Pimcore 11 only 😉
